packages/kotti-ui/source/kotti-accordion/hooks.ts

```diff
@@ -0,0 +1,53 @@
+import { ref, Ref } from '@vue/composition-api'
+export type Options = KeyframeAnimationOptions & {
+	isInitiallyClosed?: boolean
+}
+
+const getDefaultOptions = (): Options => ({
+	duration: 250,
+	easing: 'ease',
+	fill: 'forwards',
+	isInitiallyClosed: true,
+})
+
+export const useSlideAnimaion = (
+	element: Ref<HTMLElement | null>,
+	options: Options,
+) => {
+	const { isInitiallyClosed, ...restOptions } = options
```

Comment:
i think u can just skip this entire body if element is null

so u can start with an `if` condition here, if it's null, just return `{ isContentOpen: false, toggle: () => {})`

instead of all `element.value?.` you got there